src/Content/Version.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+namespace Kirby\Content;
+
+use Kirby\Cms\ModelWithContent;
+use Kirby\Exception\NotFoundException;
+
+/**
+ * The Version class handles all actions for a single
+ * version and is identified by a VersionId instance
+ *
+ * @internal
+ * @since 5.0.0
+ *
+ * @package   Kirby Content
+ * @author    Bastian Allgeier <bastian@getkirby.com>
+ * @link      https://getkirby.com
+ * @copyright Bastian Allgeier
+ * @license   https://getkirby.com/license
+ */
+class Version
+{
+	public function __construct(
+		protected ModelWithContent $model,
+		protected VersionId $id
+	) {
+	}
+
+	/**
+	 * Returns a Content object for the given language
+	 */
+	public function content(string $language = 'default'): Content
+	{
+		return new Content(
```

Comment:
I assume `Version` will later cache the content object for each language, right?

Reply:
Yes, exactly. But this isn't implemented in the other PRs so far. It's only on the todo list in Notion